### DISCUSSION TITLE
Analog led not working on 1.8.2

### DISCUSSION THREAD

**Discussion by Snkfear** [darthcloud/BlueRetro#591] [Q&A] [answered]
https://user-images.githubusercontent.com/17600591/215604888-cd7f2600-1281-48b8-aed0-843615ad8c35.mp4

I've updated my BlueRetro PS1/2 Receiver by mi213 (https://github.com/Micha213/BlueRetro-PS1-2-Receiver) from 1.7.1 to 1.8.2 HW1 and now the blue led that before indicated the "ANALOG" status emulating the led on a PS2 controller now doesn't power on.

Using a DualShock 4, when the controller gets paired the blue led remains turned off and if I push the PS button to toggle on/off the analog mode, it just doesn't work.

**Comment by darthcloud** [comment]
I removed analog LED support in v1.8

It conflicted with the new port LED feature.

**Reply by darthcloud** [reply]
The plan in the future would be to used LEDs of the controller for that.

Like Mute LED on PS5, change LED intensity on PS4 or Home LED on switch.

But if using any other controller the feature would be lost.

**Comment by darthcloud** [comment, marked as answer]
After reflection I will bring the feature back as it was for HW1 firmware.

For HW2 I might put it on the port 3 & 4 LED as they are not used anyway.

**Reply by Snkfear** [reply]
Thank you very much! I know that in a project like this some features may be discarded in order to implement new ones, so it's nice to have this back.

**Comment by Micha213** [comment]
I just updated from 1.7.3 to 1.8.2 and noticed that the analogue LED from the PS1-2 receiver no longer works. It's only a Playstation specific feature, but I think it's quite useful. I think it's great when it's implemented again. 🙏😊

**Comment by Micha213** [comment]
> and if I push the PS button to toggle on/off the analog mode, it just doesn't work.

Switching analog mode on and off still works. Only it is currently no longer displayed via the blue LED.
